tools/perf/ucc_pt_coll_allgather.cc

```diff
@@ -52,16 +53,19 @@ ucc_status_t ucc_pt_coll_allgather::init_coll_args(size_t count,
     return st;
 }
 
+float ucc_pt_coll_allgather::get_bw(float time_ms, int grsize,
+                                    ucc_coll_args_t args)
+{
+    float N = grsize;
+    float S = N * args.dst.info.count * ucc_dt_size(args.dst.info.datatype);
```

Comment:
why "count" is multiplied by N here in calculations?

Reply:
receive buffer size is N * args.dst.info.count * ucc_dt_size(args.dst.info.datatype), isn't it?